### ISSUE TITLE
pulseaudio fails to link

### ISSUE THREAD

**Issue by iMichka** [Homebrew/linuxbrew-core#16842]
# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.

- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

## What you were trying to do (and why)

brew install pulseaudio (on CI)

## What happened (include command output)

```
2019-10-31T22:29:30.8117524Z The formula built, but is not symlinked into /home/linuxbrew/.linuxbrew
2019-10-31T22:29:30.8117872Z Could not symlink share/glib-2.0/schemas/gschemas.compiled
2019-10-31T22:29:30.8118968Z Target /home/linuxbrew/.linuxbrew/share/glib-2.0/schemas/gschemas.compiled
2019-10-31T22:29:30.8119310Z already exists. You may want to remove it:
2019-10-31T22:29:30.8119788Z   rm '/home/linuxbrew/.linuxbrew/share/glib-2.0/schemas/gschemas.compiled'
2019-10-31T22:29:30.8120022Z 
2019-10-31T22:29:30.8120209Z To force the link and overwrite all conflicting files:
2019-10-31T22:29:30.8120631Z   brew link --overwrite pulseaudio
2019-10-31T22:29:30.8120858Z 
2019-10-31T22:29:30.8121038Z To list all files that would be deleted:
2019-10-31T22:29:30.8121497Z   brew link --overwrite --dry-run pulseaudio
2019-10-31T22:29:30.8121885Z 
2019-10-31T22:29:30.8122026Z Possible conflicting files are:
2019-10-31T22:29:30.8122355Z mkdir -p /home/linuxbrew/.linuxbrew/etc/pulse
2019-10-31T22:29:30.8122728Z /home/linuxbrew/.linuxbrew/share/glib-2.0/schemas/gschemas.compiled
2019-10-31T22:29:30.8123102Z mkdir -p /home/linuxbrew/.linuxbrew/share/zsh
2019-10-31T22:29:30.8123466Z mkdir -p /home/linuxbrew/.linuxbrew/share/zsh/site-functions
```

**Comment by iMichka** [comment]
https://stackoverflow.com/questions/56309056/is-gschemas-compiled-architecture-specific-can-i-ship-it-with-my-python-library

```
You should not ship a compiled gschemas.compiled cache as part of your app — instead, you should just ship your *.gschema.xml file. gschemas.compiled is intended as a system-wide cache of all the *.gschema.xml files in (by default) /usr/share/glib-2.0/schemas, and should be regenerated by the package manager after an app installs a new schema.

Debian has a lintian check to prevent packages shipping gschemas.compiled, for example.
```

**Comment by iMichka** [comment]
Fixed by #18253
